### PR TITLE
Automatic undo transactions that have not been confirmed for too long

### PR DESCRIPTION
**Pull request by fxsjy**
#80

### CODE REVIEW

**Comment by ToWorld** [comment]
> When the system rolls back transactions, it will roll back this more than 300 seconds, instead of triggering the rollback in 300 seconds. Is that right?

It marks unconfirmed transaction over 300 seconds and undoes the transaction, which is same as the flag of hasConflict, when doing the function of processUnconfirmTxs.

File: utxo/utxo.go [hunk 1]
```diff
@@ -784,7 +788,11 @@ func (uv *UtxoVM) sortUnconfirmedTx() (map[string]*pb.Transaction, TxGraph, erro
 	var totalDelay int64
 	now := time.Now().UnixNano()
 	for txID, tx := range txMap {
-		totalDelay += (now - tx.Timestamp)
+		txDelay := (now - tx.Timestamp)
```
Comment (yucaowang) [inline utxo/utxo.go]: Just curious: If someone post a Tx with a fake timestamp which is ahead of current time for thousands of seconds, it the Tx will be pending in unconfirm Tx?  If so, how we can deal with such Tx using delay time?

Comment (fxsjy) [reply utxo/utxo.go]: yes, that is a bug. I have opened another issue: #92

Comment (yucaowang) [reply utxo/utxo.go]: I see

**Pull request merged**